testng-eclipse-plugin/src/main/org/testng/eclipse/ui/TestNGQuickFixProcessor.java

```diff
@@ -76,7 +77,7 @@ private List getAddTestNGToBuildPathProposals(IInvocationContext context,
       if (maybeTestNGPackage(s)) { 
         proposals.add(new TestNGAddLibraryProposal(context, 11));
       }
-      if (maybeTestNGAnnotation(s) && isAnnotation(context)) { 
+      if (maybeTestNGAnnotation(s) && isAnnotation(context, location)) { 

```

Comment:
i know this add testng lib proposal rarely used today since maven is the de-fact tool for dependency management.
however, this is to fix a rare case as well, that if people copy the project source code and import to eclipse, there's missing testng lib, if people click the quick fix, the add lib proposal does not present.